README.md

```diff
@@ -67,6 +67,13 @@ To remove the server and clear the volumes of database data, run the command:
 ```
 docker compose down --volumes
 ```
+After a successful docker installation and running the services, The Docker will also creates a seed data for you and also will create an admin dashboard account.
```

Comment:
This is great, just some suggestions to tighten up the wording a bit.

How about:
```
After a successful docker installation, an admin dashboard account is automatically created.
The default username...
```

Reply:
Okay, Will update accordingly,